src/clr-addons/quick-list/add-option.html

```diff
@@ -1,14 +1,12 @@
-<div class="quick-list-input">
-  <clr-select-container>
-    <select class="quick-list-select" clrSelect name="options" [(ngModel)]="selectedValue">
-      <option *ngIf="blankOption.id === selectedValue" [selected]="true" [disabled]="true" [ngValue]="blankOption.id">
-        {{blankOption.label}}
-      </option>
-      <option *ngFor="let val of getValues()" [selected]="val.id === selectedValue" [ngValue]="val.id">
-        {{val.label}}
-      </option>
-    </select>
-  </clr-select-container>
+<div class="clr-select-wrapper quick-list-input">
```

Comment:
is something wrong with using clr-select-container component and clrSelect directive?
if not please use them, so no adaption is needed if any css class is renamed

Reply:
the select container is unusable for this component (content is no longer shown)